target_snowflake/__init__.py

```diff
@@ -523,6 +544,10 @@ def main():
 
     # Consume singer messages
     singer_messages = io.TextIOWrapper(sys.stdin.buffer, encoding='utf-8')
+    
+    LOGGER.info("**PR** LINE 527 singer_messages")
+    
+    LOGGER.info("Exiting normally")
```

Comment:
Delete unneeded log.

Reply:
Created a new branch and this one no longer exists.